Sources/Datadog/Datadog.swift

```diff
@@ -95,10 +95,8 @@ public class Datadog {
                 consolePrint("⚠️ Overriding RUM debugging due to \(LaunchArguments.DebugRUM) launch argument")
                 Datadog.debugRUM = true
             }
-        } catch let error as ProgrammerError {
-            consolePrint("\(error)")
         } catch {
-            defaultDatadogCore = NOOPDatadogCore()
+            consolePrint("\(error)")
```

Comment:
I've revert this so we have a console print on every failure. The `defaultDatadogCore` is set at the very end of the initialisation.